Add active executions. (#15057)
cockroachdb/docs: _includes/v22.2/ui/active-transaction-executions.md

## Active Executions table

Click <img src="{{ 'images/common/ui-columns-button.png' | relative_url }}" alt="Column selector" /> to select the columns to display in the table.

The Active Executions table gives details for each transaction fingerprint in the transaction:

Column | Description
-----|------------
Transaction Execution ID | The execution ID of the transaction.
Most Recent Statement | The most recently executed statement in the transaction.

> Status | The status of the execution: Preparing, Waiting, or Executing.

Reviewer: Status names should be in backticks.

Author: Fixed.

> Start Time (UTC) | The time the execution started.

Reviewer: Same comment as above re: timestamp.

Author: Fixed.

Reviewer: This looks unchanged on my end (still reads "The time the...")

Elapsed Time | The time elapsed since the transaction started.

> Time Spent Waiting | The time the execution spent waiting for access to resources.

Reviewer 2: Same comment as above on lock contention

Author: Fixed in latest commit.

> Time Spent Waiting | The time the execution spent waiting and experiencing lock contention.

Reviewer: Is this accurate?

```suggestion
Time Spent Waiting | The amount of time the execution experienced lock contention.
```

Statements | The number of statements in the transaction.

> Retries | The number of times statements are retried.

Reviewer: Making a guess

```suggestion
Retries | The number of times statements in the transaction were retried.
```

Application | The name specified by the [`application_name`](show-vars.html#supported-variables) session setting.

> To view details of an active transaction execution, click an execution ID in the **Transaction Execution ID** column to open the **Transaction Execution** details page.

Reviewer: ```suggestion
To view details of an active transaction execution, click an execution ID in the **Transaction Execution ID** column to open the [**Transaction Execution** details page](#transaction-execution-details-page).
```

## Transaction Execution details page

> The details page provides the following details on the transaction execution.

Reviewer: ```suggestion
The Transaction Execution details page provides the following details on the transaction execution.
```

> - **Start Time (UTC)**: the time the execution started.

Reviewer: Same comment as above re: capitalization

Author: Fixed.

- **Elapsed Time**: the time elapsed since the transaction started.

> - **Status**: the status of the execution: Preparing, Waiting, or Executing.

Reviewer: Same comment re: backticks

- **Priority**: the [priority](transactions.html#transaction-priorities) of the transaction.
- **Full Scan**: whether the execution performed a full scan of the table.
- **Internal Retries**: the number of retries of statements in the transaction.

> - **Last Retry Reason**: the reason for the last statement retry.

Reviewer: Is there a small, limited number of possible reasons? If so, perhaps list them here, if they have succinct names?

> - **Last Retry Reason**: The reason for the last statement retry.

Author: @maryliag Is there a list of reasons I can provide?

Reviewer 2: We can link to the following doc retry errors: https://www.cockroachlabs.com/docs/stable/transaction-retry-error-reference.html

- **Number of Statements**: the number of statements in the transaction.
- **Application Name**: the name specified by the [`application_name`]({{ link_prefix }}show-vars.html#supported-variables) session setting.
- **Most Recent Statement Execution ID**: Link to the ID of the most recently [executed statement](ui-statements-page.html#active-executions-table) in the transaction.

> - **Session ID**: Link to the [session](ui-sessions-page.html) in which the transaction is running.

Reviewer: ```suggestion
- **Session ID**: Link to the ID of the [session](ui-sessions-page.html) in which the transaction is running.
```
